EPA/professional_discussion.md

```diff
@@ -0,0 +1,122 @@
+# Professional Discussion
+
+* Placeholder for Table Of Contents - Don't remove
+{:toc}

```

Comment:
I think that the structure below is reasonable with this table of contents, but I wonder whether it would better to split out the questions from the criteria, so that it's easy to just get a list of questions? We could be clever and add a tool that randomly selects ~10 split across the different criteria or something, but that can be a nice-to-have for right now!